### PR TITLE
docs(router): add guide for nested redirect routes

### PR DESCRIPTION
**Pull request by markostanimirovic**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [ ] vite-plugin-angular
- [ ] vite-plugin-nitro
- [ ] astro-angular
- [ ] create-analog
- [x] router
- [ ] platform
- [ ] content
- [ ] nx-plugin
- [ ] trpc

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/docs-app/docs/features/routing/metadata.md [hunk 1]
```diff
@@ -34,9 +33,7 @@ export default class AboutPageComponent {
 
 Routes can be defined for only the purpose of redirecting to another route.
```
Comment (goetzrobin) [inline apps/docs-app/docs/features/routing/metadata.md:34]:
```suggestion
Routes can be defined for the sole purpose of redirecting to another route.
```

File: apps/docs-app/docs/features/routing/metadata.md [hunk 2]
```diff
@@ -48,7 +45,38 @@ export const routeMeta: RouteMeta = {
 };
 ```
 
-> An exported route component is not necessary when defining a redirect
+The example above is a redirect to the `/home` route from `/` route.
```
Comment (goetzrobin) [inline apps/docs-app/docs/features/routing/metadata.md:48]:
```suggestion
The example above is a redirect from the `/` route to the `/home` route.
```

File: apps/docs-app/docs/features/routing/metadata.md [hunk 3]
```diff
@@ -48,7 +45,38 @@ export const routeMeta: RouteMeta = {
...
+src/
+└── app/
+    └── pages/
+        └── cities/
+            ├── index.page.ts
+            ├── new-york.page.ts
+            └── san-francisco.page.ts
+```
+
+and the `routeMeta` definition in the `src/app/pages/cities/index.page.ts` file:
```
Comment (goetzrobin) [inline apps/docs-app/docs/features/routing/metadata.md:68]:
```suggestion
and the following `routeMeta` definition to the `src/app/pages/cities/index.page.ts` file:
```

File: apps/docs-app/docs/features/routing/metadata.md [hunk 4]
```diff
@@ -48,7 +45,38 @@ export const routeMeta: RouteMeta = {
...
+```ts
+import { RouteMeta } from '@analogjs/router';
+
+export const routeMeta: RouteMeta = {
+  redirectTo: '/cities/new-york',
+  pathMatch: 'full',
+};
+```
+
+navigating to `/cities` will redirect to `/cities/new-york`.
```
Comment (goetzrobin) [inline apps/docs-app/docs/features/routing/metadata.md:79]:
```suggestion
Now, navigating to `/cities` will redirect to `/cities/new-york`. 
One final note: Nested redirects always require an absolute path.
```

**Review by goetzrobin — COMMENTED** [review]
Left a couple suggestions! Just minor wording and I wanted to add a note that for nested redirects you will need an absolute path.

**Review by goetzrobin — APPROVED** [review]

File: apps/docs-app/docs/features/routing/metadata.md [hunk 5]
```diff
@@ -48,7 +45,44 @@ export const routeMeta: RouteMeta = {
...
```
Comment (brandonroberts) [inline apps/docs-app/docs/features/routing/metadata.md:79]:
```suggestion
Navigating to `/cities` redirects to `/cities/new-york`.
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**